src/htcondor_es/convert_to_json.py

```diff
@@ -603,16 +603,12 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
     result['ScheddName'] = ad.get("GlobalJobId", "UNKNOWN").split("#")[0]
 
     # Determine type
-    analysis = ("CRAB_Id" in ad) or (ad.get("AccountingGroup", "").startswith("analysis."))
+    if cms:
+        result["Type"] = ad.get("CMS_Type", "unknown").lower()
+    analysis = (result.get("Type") == 'analysis')
```

Comment:
This may break if result dict will not have "Type" key. Better to use
```
analysis = (result.get("Type", None) == "analysis")
```

Reply:
I think both are equivalent, as the default ['default' value for `get` is None](https://docs.python.org/3/library/stdtypes.html#dict.get).